src/app/educate-teacher/my-classes/components/resources/resources.component.scss

```diff
@@ -54,6 +62,16 @@
     .not-availble {
       text-align: center;
     }
+
+    .errorBox {
+      padding: 10px;
+      color: #ff004c;
```

Comment:
This is minor suggestion. 
`padding: 10px` doesn't require `if` your are writing `appCardBodyStyle` without any arguments.

Reply:
no, i need 10px, so 23 +10 = 33; and 23 is because resources have 10px margin